README.md

```diff
@@ -6,12 +6,13 @@
 
 - css
 - html
+- Javascript
 
 ## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.